File: csharp/Exercises/OutputAndOperators/ex004.cs
//Instructions: Write a program that displays the integer part of a number entered by the user on the screen.

namespace Exercises.OutputAndOperators
{
    public static class Ex004
    {
        public static void Run()
        {
            float r;
            Console.WriteLine("Enter a real number ");
            r = Convert.ToSingle(Console.ReadLine()); //Please note that in a Portuguese Windows console, the decimal separator is a comma.
            Console.WriteLine("The integer part of {0} is {1}",r,(int)r);
        }
    }
}

File: javascript/OutputAndOperators/ex004.js
//Instructions: Write a program that displays the integer part of a number entered by the user on the screen.

//Importing the readline
const readline = require('readline/promises');

//Creating the reading interface
const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout
});

async function run() {
  //Asking the user a question
  const answer = await rl.question("Enter a real number: ");
  //Converting the answer to a number
  const r = parseFloat(answer);
  
  //Taking the whole part
  console.log(`The integer part of ${r} is ${Math.trunc(r)}`);

  //Closing the interface
  rl.close();
}

run();
